wheel/src/run_generator.rs

```diff
@@ -142,11 +144,46 @@ pub fn run_generator(
                 None,
                 Some(convert_spend_bundle_conds(&allocator, spend_bundle_conds)),
             ))
-        }
+        },
         Ok((error_code, _)) => {
             // a validation error occurred
             Ok((error_code.map(|x| x.into()), None))
         }
         Err(eval_err) => eval_err_to_pyresult(py, eval_err, allocator),
     }
 }
+
+#[pyfunction]
+#[allow(clippy::borrow_deref_ref)]
```

Comment:
This is good, gotta steal it.

Reply:
Hmm, it didn't help me in my case. Does it actually work here?

Reply:
iirc, this was to suppress a buggy clippy warning. I can't reproduce it now, so it's possible it's been fixed. I'll remove both of these